.gitignore

```diff
@@ -4,3 +4,6 @@
 
 # OSX trash
 .DS_Store
+
+#apiserver local up cert trash
```

Comment:
Adding a whitespace looks better, but of course this is not very important.
```
# apiserver local up cert trash
```

Reply:
Good suggestion 
I will fix it